worker/buildbot_worker/test/fake/reactor.py

```diff
@@ -83,7 +83,7 @@ def run(self) -> None:
     def running(self) -> bool:
         raise NotImplementedError()
 
-    def resolve(self, name: str, timeout: Sequence[int]) -> defer.Deferred[str]:
+    def resolve(self, name: str, timeout: Sequence[int] = (1, 3, 11, 45)) -> defer.Deferred[str]:
```

Comment:
We should ignore this line in terms of typing because mypy will fail if other version of Twisted is installed (e.g. on developer machine).